SampleApp/BaseUI.swift

```diff
@@ -48,7 +48,9 @@ extension NSLayoutConstraint {
                                 let duration = userInfo[UIWindow.keyboardAnimationDurationUserInfoKey] as? TimeInterval {
                                 view?.layoutIfNeeded()
                                 UIViewPropertyAnimator(duration: duration, curve: curve, animations: {
-                                    self.constant = -finalFrame.height
+                                    let padding: CGFloat = 10
```

Comment:
While it's currently only used once, we probably don't want this to add padding for every view that happens to use it. Better to add the additional padding as an optional argument to this function than hard code it.

Reply:
let me take a look again

Reply:
wontfix